src/components/main/index.tsx

```diff
@@ -121,10 +273,14 @@ export default function MainPage({ title, favicon, currentInfos, signature }: Ma
                     <Tab label="Logs" {...a11yProps(1)} />
                 </Tabs>
                 <TabPanel value={value} index={0}>
-                    <Button className={classes.seeMore} startIcon={<FullscreenIcon />}>
+                    <Button className={classes.seeMore} startIcon={<FullscreenIcon />} onClick={handleClickOpen}>
                         Full Screen
                     </Button>
-                    <MembershipListPage />
+                    <Dialog open={open} fullScreen onClose={handleClose} TransitionComponent={Transition}>
+                        <DialogHeader onClose={handleClose} />
+                        <MembershipListPage memberships={dummyMemberships} />    
+                    </Dialog>
+                    <MembershipListPage memberships={dummyMemberships} />
```

Comment:
Appearance.
Needed to check if it also shows a browser-sized full screen or not.
<img width="1920" alt="스크린샷 2020-09-02 오후 5 48 29" src="https://user-images.githubusercontent.com/14817195/91960165-84cfee80-ed44-11ea-9765-a4506b8b78b1.png">